### PR TITLE
Add React/Vue snippets for Chart/Range Selector zooming and panning topics

### PR DESCRIPTION
**Pull request by dxBeardedBear**

### CODE REVIEW

File: concepts/05 Widgets/Chart/95 Zooming and Panning/10 Using the RangeSelector Widget.md [hunk 1]
```diff
@@ -66,6 +67,103 @@ The **Chart** can be zoomed and panned using the [RangeSelector](/concepts/05%20
...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template> 
+        <DxChart
+            :data-source="data"
+            :series="seriesConfiguration"
+        >
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/95 Zooming and Panning/10 Using the RangeSelector Widget.md]:
```suggestion
            :series="seriesConfiguration">
```

File: concepts/05 Widgets/Chart/95 Zooming and Panning/10 Using the RangeSelector Widget.md [hunk 2]
```diff
@@ -66,6 +67,103 @@ The **Chart** can be zoomed and panned using the [RangeSelector](/concepts/05%20
...
+            <DxArgumentAxis :visual-range="chart_visualRange"/>
+        </DxChart>
+        <DxRangeSelector
+            :data-source="data"
+            :value.sync="chart_visualRange"
+        >
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/95 Zooming and Panning/10 Using the RangeSelector Widget.md]:
```suggestion
            :value.sync="chart_visualRange">
```

File: concepts/05 Widgets/Chart/95 Zooming and Panning/10 Using the RangeSelector Widget.md [hunk 3]
```diff
@@ -66,6 +67,103 @@ The **Chart** can be zoomed and panned using the [RangeSelector](/concepts/05%20
...
+            this.state = { chart_visualRange: [ ... ] };
+            this.updateChartVisualRange = () => this.setState({ chart_visualRange: e.value });
+        }
+
+        render() {
+            return (
+                <Chart
+                    dataSource={data}
+                    series={seriesConfiguration}
+                >
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/95 Zooming and Panning/10 Using the RangeSelector Widget.md]:
```suggestion
                    series={seriesConfiguration}>
```

File: concepts/05 Widgets/Chart/95 Zooming and Panning/10 Using the RangeSelector Widget.md [hunk 4]
```diff
@@ -66,6 +67,103 @@ The **Chart** can be zoomed and panned using the [RangeSelector](/concepts/05%20
...
+                    <ArgumentAxis visualRange={this.state.chart_visualRange} />
+                </Chart>
+                <RangeSelector
+                    dataSource={data}
+                    onValueChanged={this.updateChartVisualRange}
+                >
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/Chart/95 Zooming and Panning/10 Using the RangeSelector Widget.md]:
```suggestion
                    onValueChanged={this.updateChartVisualRange}>
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/Chart/95 Zooming and Panning/10 Using the RangeSelector Widget.md (hunk 1 above)
Comment (dxBeardedBear) [reply concepts/05 Widgets/Chart/95 Zooming and Panning/10 Using the RangeSelector Widget.md]: Fixed

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**